core/src/main/ruby/jruby/truffle/core/float.rb

```diff
@@ -8,8 +8,18 @@
 
 class Float
 
-  NAN = 0.0 / 0.0
-  INFINITY = 1.0 / 0.0
-  EPSILON = 2.2204460492503131e-16
+  NAN        = 0.0 / 0.0
+  INFINITY   = 1.0 / 0.0
+  EPSILON    = 2.2204460492503131e-16
+  RADIX      = 2
+  ROUNDS     = 1
+  MIN        = 2.2250738585072014e-308
```

Comment:
Ok, I'll keep my eye out for problems and we can set the constants in Java code if needs be. Let's not change it until we've seen a problem though. I think Ruby code should be the default unless there is a good reason otherwise.